src/client/commands/submit.rs

```diff
@@ -157,6 +163,15 @@ fn validate_name(name: String) -> anyhow::Result<String, anyhow::Error> {
     }
 }
 
+fn get_validated_path(cwd: String) -> anyhow::Result<PathBuf, anyhow::Error> {
+    let working_dir = PathBuf::from(cwd);
+    match fs::metadata(&working_dir) {
+        Ok(cwd) if cwd.is_dir() => Ok(working_dir),
+        Err(error)  => Err(anyhow!("Error setting cwd, {}", error)),
```

Comment:
I would remove this line and just keep the fallback ("The path supplied is not a directory").

Reply:
I was thinking about possible errors due to permissions issue, removed

Reply:
We can't solve all the issues for the user, let him also do some debugging too :D

Btw @spirali it might happen that the passed cwd is not valid on the node of the submitter, but it will be valid on the node of the worker. Do we want to support this case?

Reply:
Definitely, it should be not be an error that directory does not exists in submitter machine (as it may exists only on remote side). Or at least it should not be a hard error and only a warning. It is probably not a problem now, but it will be a problem in cloud partition.

Reply:
Let's keep it as a warning with a message like `<cwd> is not a valid directory on the current node.`.